curations/pypi/pypi/-/regex.yaml

```diff
@@ -15,6 +15,9 @@ revisions:
   2020.4.4:
     licensed:
       declared: Python-2.0
+  2020.5.14:
+    licensed:
+      declared: PSF-2.0
```

Comment:
PSF-2.0 is a rarer thing and is a subset of the complete Python-2.0 license used solo.
Here regex is based on, includes and is derived from the main re Python code and is therefore under the same terms, so Python-2.0 and not the bare PSF